### PR TITLE
Feature/3 learning UI

### LINKED ISSUE #3: design sorting/learning UI and make working demo (with dummy data). if the classification is correct, the next image is displayed

### PR DESCRIPTION
**Pull request by Nitwix**
# This PR adds the Learning UI 

With the added LearningActivity, the learner can drag and drop a colored square onto the corresponding colored square. (the colored squares will be replaced by images in the end). 
 What remains to be done for it to be a proper PR:

- [x] Add automated UI tests for the learning UI

### CODE REVIEW

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningTest.kt [hunk 1]
```diff
@@ -1,49 +1,98 @@
...
-        override fun getConstraints(): Matcher<View> {
-            TODO("Not yet implemented")
-        }
-
-        override fun getDescription(): String {
-            TODO("Not yet implemented")
+    @Test fun dragImageOnCorrectCategory(){
+        val imToSort = getImageToSort()
+        val startDescr = imToSort.contentDescription
+        val NUMBER_OF_TENTATIVES = 100
```
Comment (bbjubjub2494) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningTest.kt]:
```suggestion
        val NUMBER_OF_ATTEMPTS = 100
```
"tentative" is an improper gallicism :fr: in English, _tentative_  is an adjective. _attempts_ is a better alternative

Comment (Nitwix) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningTest.kt]: indeed, I didn't remember the proper english translation. Thanks

Comment (bbjubjub2494) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningTest.kt]: (also the comment)

You're welcome

**Review by NicolasVial — APPROVED** [review]
It looks good to me. I didn't find anything that I had a problem with.

**Review by bbjubjub2494 — APPROVED** [review]
LGTM overall. Could you
- autoformat the 3 Kotlin files
- rebase and squash a bit

**Pull request merged**